_posts/2019-01-25-bill-israel.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "PyTN 2019 Meet Our Organizers: Bill Israel"
+published: true
+categories: [blog, tech, conference]
+tags: [python, tech, conference, tennessee, nashville]
+---
+Bill Israel in the main organizers of PyTN. Whether you're speaking, sponsoring, or attending, he's been the one holding our ship together. Get to know him better and his opinions on the conference, Nashville, and more. He's a pretty rad guy.
```

Comment:
I think it's simpler here to just say "Bill is the Chair of PyTN 2019."

Reply:
Done!